src/WCSimSteppingAction.cc

```diff
@@ -150,7 +150,7 @@ void WCSimSteppingAction::Distortion(G4double x,G4double y)
 }
 
 
-double WCSimSteppingAction::FieldLines(G4double x,G4double y,G4int coord)
+double WCSimSteppingAction::FieldLines(G4double /*x*/,G4double /*y*/,G4int /*coord*/)
```

Comment:
Same comment as above. x,y, and coord are commented out in this PR. It should be left the same unless there is a reason to change it for this PR.